subsys/bluetooth/mesh/shell/shell_sensor_cli.c

```diff
@@ -42,6 +42,8 @@ static void descriptors_print(const struct shell *shell, int err,
 
 static int cmd_desc_get(const struct shell *shell, size_t argc, char *argv[])
 {
+	int err = 0;
```

Comment:
Having this all the way up here confused me, I had to read the `return err` on line 60 to be sure that it was still zero when arg parsing started. I'd suggest to move this to after the `if (argc == 1) { ... }`.

Reply:
I dont think it can be placed after `if (argc == 1) { ... }` unless you want me to declare it twice. `err` is used both inside the `if (argc == 1) ` body and later.  I can move it to right before the `if (argc == 1) { ... }`?

Reply:
Oh, sorry, I meant

```
int err;

...

if (argc == 1) {
    ...
}

struct bt_mesh_sensor_descriptor rsp;

err = 0;
uint32_t sensor_id = shell_strtoul(argv[1], 10, &err);
...
```

Reply:
Ahh... I see 👍